ConnectorRhino/ConnectorRhino/ConnectorRhinoShared/UI/ConnectorBindingsRhino.Receive.cs

```diff
@@ -335,7 +335,7 @@ ApplicationObject CreateApplicationObject(Base current, string containerId)
       ApplicationObject NewAppObj()
       {
         var speckleType = current.speckle_type
-          .Split(new[] { ':' }, StringSplitOptions.RemoveEmptyEntries)
+          .Split(new[] { '.' }, StringSplitOptions.RemoveEmptyEntries)
```

Comment:
Just want to confirm that this is intentional. Didn't this split the speckle_type on `:` so that we'd get the last class entry with it's entire namespace? i.e. `Objects.BuiltElements.Beam:Objects.BuiltElements.Revit.RevitBeam` became `Objects.BuiltElements.Revit.RevitBeam`, but with this change it becomes just `RevitBeam`.

Reply:
yah, some users have given feedback that the namespace is too much info